src/services

```diff
@@ -561,7 +561,7 @@ p({
     dao = new foam.dao.KeyValueDAO.Builder(x)
       .setDelegate(new foam.dao.NullDAO(x, foam.nanos.fs.File.getOwnClassInfo()))
       .build();
-
+    dao = new foam.nanos.fs.SupportFileTypeDAO(x, dao);
```

Comment:
why is this on demoObjectDAO? 
Should this not decorate the FileDAO stack?

Reply:
this is on FileDAO stack, not demoObjectDAO

Reply:
ok, last thing, can you move this to fs/services.jrl as well please.  Realize you are just updating, but since we are moving stuff around this is the moment. Thanks.   It just helps down the road with less merge conflicts when the nspecs are in separate service files rather than one big one.